packages/common-components/src/Tabs/Tabs.tsx

```diff
@@ -75,7 +76,7 @@ const Tabs = <TabKey, >({ className, children, activeKey, injectedClass, onTabSe
                   clsx(
                     elem.props.tabKey === activeKey && "selected",
                     classes.tabBar,
-                    injectedClass?.tabBar
+                      injectedClass?.tabBar
```

Comment:
Tiny nit, committing right away to merge
```suggestion
                    injectedClass?.tabBar
```